www/js/components/MetricDetails.tsx

```diff
@@ -0,0 +1,87 @@
+
+import React, { useRef, useState } from 'react';
+import { array, string, bool } from 'prop-types';
+import { angularize } from '../angular-react-helper';
+import { View } from 'react-native';
+import { useTheme } from 'react-native-paper';
+
+const MetricsDetails = ({ chartData}) => {
+ 
+  const { colors } = useTheme();
+  const [ numVisibleDatasets, setNumVisibleDatasets ] = useState(1);
+
+  const defaultPalette = [
```

Comment:
I'm guessing this was copied over from `BarChart`. It doesn't need to be here though

Reply:
If you want to copy the contents of a file as a starting point for a new file, that's ok -- just make sure you remove the bits that aren't relevant.
we don't want unused code lying around